ape_hardhat/providers.py

```diff
@@ -382,9 +382,25 @@ def get_transaction_trace(self, txn_hash: str) -> Iterator[TraceFrame]:
         Returns:
             Iterator(TraceFrame): Transaction execution trace object.
         """
-        logs = self._make_request("debug_traceTransaction", [txn_hash]).structLogs
-        for log in logs:
-            yield TraceFrame(**log)
+
+        for frame in self._make_request("debug_traceTransaction", [txn_hash]).structLogs:
+            yield TraceFrame(**frame)
+
+    def get_calltree_from_trace(
+        self, txn_hash: str, show_internals=False, **root_node_kwargs
```

Comment:
Does `show_internals` not get used?

Reply:
Based on the evm-trace PR, it seems we should be passing it in the `get_calltree_from_trace` call